.github/pull_request_template.md

```diff
@@ -2,18 +2,28 @@
 
 ## Related issue number (if any)
 
-## Loom demo (if any)
-
-## Checklist before requesting a review
-- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
-- [ ] I have performed a self-review of my code.
-- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
-- [ ] If this is a new feature, I have added unit tests and integration tests.
-- [ ] I have run the integration tests locally and they are passing.
-- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
-- [ ] All features on the UI continue to work correctly.
-- [ ] Added one of the following CI labels:
```

Comment:
We can't add this until review is done right, integration tests before rest of review will consume resources that has cost associated. So description can mention do this once code review is completed.